### PR TITLE
manage user

### PR DESCRIPTION
**Pull request by minhquang31011998**
## Related Tickets:
- #29512 https://edu-redmine.sun-asterisk.vn/issues/29512

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by minhquang31011998** [comment]
Screenshot from 2020-10-14 13-10-22

Screenshot from 2020-10-14 13-10-24

**Comment by minhquang31011998** [comment]
ready

File: app/Http/Controllers/Backend/UserController.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        if ($request->has('sort')) {
+            if ($request->get('sort') == trans('name')) {
+                $users = $users->orderByRaw('name ASC');
+            } elseif ($request->get('sort') == trans('date_created')) {
+                $users = $users->orderByRaw('created_at DESC');
+            } elseif ($request->get('sort') == trans('active')) {
+                $users = $users->where('is_active', '=', config('config.status_active'));
+            } elseif ($request->get('sort') == trans('hidden')) {
+                $users = $users->where('is_active', '=', config('config.status_hidden'));
+            }
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Backend/UserController.php]: dùng switch case

File: resources/assets/backend/js/cruduser.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+$('#status').on('click', function () {
+    $.ajax({
+        type: "POST",
+        url: '/admin/user/changeStatus/' + $('#userId').val(),
+        dataType: 'json',
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        },
+        success: (response) => {
+            if (response.is_active == 1) {
```
Comment (huyenttm-0901) [inline resources/assets/backend/js/cruduser.js]: config số 1 đi em

**Comment by minhquang31011998** [comment]
ready

**Pull request merged**